service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentsValidatorServiceTest.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentSocialWorkOther;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.UUID;
+import javax.validation.Validation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+class DocumentsValidatorServiceTest {
+    private DocumentsValidatorService documentsValidatorService;
+    private static final String STATUS_ATTACHED = "Attached";
+    private static final String STATUS_SWET = "Included in social work evidence template (SWET)";
+    private static final DocumentReference POPULATED_DOCUMENT = DocumentReference.builder()
+        .filename("Mock filename")
+        .build();
+
+    @BeforeEach()
+    private void setup() {
+        documentsValidatorService = new DocumentsValidatorService(Validation
+            .buildDefaultValidatorFactory()
+            .getValidator());
+    }
+
+    @Test
+    void shouldGenerateErrorsWhenAdditionalDocumentsAreMissingDocumentTitle() {
+        List<Element<DocumentSocialWorkOther>> additionalDocuments = createSocialWorkOtherDocuments("");
+        List<String> validationErrors = documentsValidatorService.validateSocialWorkOtherDocuments(additionalDocuments);
+        assertThat(validationErrors).containsOnlyOnce("You must give additional document 1 a name.",
+            "You must give additional document 2 a name.");
```

Comment:
Why we have got two message if only one document is added?

Reply:
`createSocialWorkOtherDocuments` Will create two additional documents.